recipes/tensorflow-text/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set version = "2.17.0" %}
+{% set tensorflow_version = ".".join(version.split(".")[:2]) %}
+
+package:
+  name: tensorflow-text
+  version: {{ version }}
+
+source:
+  url: https://github.com/tensorflow/text/archive/v{{ version }}.tar.gz
+  sha256: 55bff24f120024664124c8fdc0ab842ab314bd5ff2791770a80f32bb67cb26d1
+
+build:
+  number: 0
+  skip: true  # [win]
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
+
+requirements:
+  build:
+  - {{ compiler('c') }}
+  - {{ stdlib('c') }}
+  - {{ compiler('cxx') }}
+  - {{ compiler('cuda') }}                 # [cuda_compiler_version != "None"]
+  - bazel >=5.2.0
+  host:
+  - python
+  - pip
+  - packaging
+  - setuptools >=70
+  - wheel
+  - absl-py
+  - tensorflow {{ tensorflow_version }}
+  - libtensorflow *=cuda*  # [cuda_compiler_version != "None"]
+  - libtensorflow *=cpu*   # [cuda_compiler_version == "None"]
+  run:
+  - python
```

```suggestion
  - python
  run_constrained:
    - tensorflow-base *=cuda*  # [cuda_compiler_version != "None"]
    - tensorflow-base *=cpu*   # [cuda_compiler_version == "None"]
```

probably good to keep the CPU/CUDA stuff pinned together. a constraint is good since it won't conflict with the automatically exported requirements.

---

I don't think we do a good job locking the cuda/cpu stuff.

---

I mean, the full tensorflow package should be a run dependency like https://github.com/conda-forge/tf-keras-feedstock/blob/main/recipe/meta.yaml, right?

---

Added a full dependency on tensorflow for now, as the pip package depends explicitly on tensorflow.